android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -49,6 +51,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         View view = super.onCreateView(inflater, container, savedInstanceState);
 
+        downloadCompleteHandler = new DownloadCompleteHandler(getContext(), R.string.menu_feed);
```

Comment:
Used its progress bar to avoid code redundancy. Want me to setup another progress bar here?